website/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Transport API Repository coverage map</title>
+    <meta charset='utf-8'>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel='stylesheet' href='https://unpkg.com/maplibre-gl@latest/dist/maplibre-gl.css' />
+    <script src='https://unpkg.com/maplibre-gl@latest/dist/maplibre-gl.js'></script>
+    <style>
+        body { margin: 0; padding: 0; }
+        html, body, #map { height: 100%; }
+        .maplibregl-popup {
+            max-width: 500px;
+        }
+    </style>
+</head>
+<body>
+<div id="map"></div>
+<script>
+    const map = new maplibregl.Map({
+        container: 'map',
+        style: {
+            version: 8,
+            sources: {
+                bg: {
+                    attribution: '<a href="https://www.openstreetmap.org/copyright">© OpenStreetMap contributors</a>',
+                    type: 'raster',
+                    tiles: ['https://tile.openstreetmap.org/{z}/{x}/{y}.png'],
```

Comment:
We could also use vector tiles. What do you think?

```suggestion
                    tiles: ['https://vector.openstreetmap.org/$VERSION/{zoom}/{x}/{y}.mvt'],
```

Reply:
Probably the better choice, but that requires a few more changes than just swapping the source URL (in particular this needs a proper (external) map style then, which means the ad hoc setup for our overlay would need to be redone differently I think). Not sure that's worth the effort for now, given this is mostly meant as a simple diagnostics/QA tool.